engineering-toolkit/eslint-config/src/typescript.js

```diff
@@ -61,9 +61,10 @@ export function typescript(config, ...overrides) {
         "@typescript-eslint/explicit-module-boundary-types": "warn",
         "@typescript-eslint/naming-convention": "off",
         "@typescript-eslint/no-empty-interface": "off",
+        "@typescript-eslint/no-empty-object-type": "off",
```

Comment:
It should be at least a warning because in 99% of cases interface like `interface Empty {}` is a mistake, such interface is redundant.

Reply:
What's the use case here, it's for extending?

Reply:
We can move this one to storefronts codebase. We have more than 20 cases of things like this:
<img width="986" alt="image" src="https://github.com/user-attachments/assets/c1576fe7-008d-465e-8413-647ef16d7ffd">
<img width="835" alt="image" src="https://github.com/user-attachments/assets/bef0ff75-e4fd-4f44-9b3f-663a05d76580">

It's code supposed to be extended by the end user (or sometimes just to create "our" own exported type - in case we need to change it in the future we don't just reexport typings from subpackages.

But as I said - if it's problematic I can move this to storefronts

Reply:
Yeah, extending, classic issue.